geotrek/api/v2/views/trekking.py

```diff
@@ -72,6 +75,28 @@ def filter_published_lang_retrieve(self, request, queryset):
             qs = qs.filter(Q(**{field_name: True}) | Q(**{field_name_parent: True}))
         return qs.distinct()
 
+    @action(detail=True)
+    def dem(self, request, pk):
+        trek = self.get_object()
+        trek_date_update = trek.get_date_update().strftime('%y%m%d%H%M%S%f')
+        json_lookup = f"altimetry_dem_area_{trek.pk}_{trek_date_update}"
+        return cached_json_response(json_lookup, trek.get_elevation_area)
+
+    @action(detail=True)
+    def profile(self, request, pk):
+        trek = self.get_object()
+        trek_date_update = trek.get_date_update().strftime('%y%m%d%H%M%S%f')
+        json_lookup = f"altimetry_profile_{trek.pk}_{trek_date_update}"
+        return cached_json_response(json_lookup, trek.get_elevation_profile_and_limits)
```

Comment:
I think you should cache elevation content, not response. use response.Response to serve cached content